docs/examples.md

```diff
@@ -238,3 +238,36 @@
           ...
           ...
 ```
+## Configure Additional Containers
+
+```
+  containers:
+        - command:
+            - bin/run-druid.sh
+            ...
+        - command:
+            - /bin/sidekick
+          image: universalforwarder-sidekick:next
+          resources:
+            requests:
+              memory: "1Gi"
+              cpu: "500m"
+            limits:
+              memory: "1Gi"
+              cpu: "500m"
+          args:
+            - -loggingEnabled=true
+          args:
+            - -loggingEnabled=true
+            - -dataCenter=dataCenter
+            ...
+          securityContext:
+            runAsUser: 506
+          imagePullPolicy: Always
+          volumeMounts:
+            - name: logstore
+              mountPath: /logstore
+          env:
+            - name: SAMPLE_ENV
+              value: SAMPLE_VALUE
+```
```

Comment:
```
additionalContainer:
    - image: universalforwarder-sidekick:next
      containerName: forwarder
      command:
        - /bin/sidekick
      imagePullPolicy: Always
      securityContext:
        runAsUser: 506
      volumeMounts:
        - name: logstore
          mountPath: /logstore
      env:
        - name: SAMPLE_ENV
          value: SAMPLE_VALUE
      resources:
        requests:
          memory: "1Gi"
          cpu: "500m"
        limits:
          memory: "1Gi"
          cpu: "500m"
      args:
        - -loggingEnabled=true
        - -dataCenter=dataCenter
        - -environment=environment
        - -application=application
        - -instance=instance
        - -logFiles=logFiles
```

Reply:
can you add a pointer stating this is scoped to cluster only.

Reply:
I added the above snippet. The one I added had a couple of copy paste errors. sorry about that.
Also added the cluster scope text to features.md. let me know what you think